rubocop.yml

```diff
@@ -173,3 +173,29 @@ Layout/FirstHashElementIndentation:
   # This one in particular forbids what Emacs does by default for multi-line
   # hash entries
   Enabled: false
+
+# Pending Cops as of 0.87.1
+Layout/EmptyLinesAroundAttributeAccessor:
+  Enabled: true
+Layout/SpaceAroundMethodCallOperator:
+  Enabled: true
+Lint/DeprecatedOpenSSLConstant:
+  Enabled: true
+Lint/MixedRegexpCaptureTypes:
+  Enabled: true
+Style/AccessorGrouping: # maybe
```

Comment:
This one may be contentious. Separating attributes can be useful to add comments. I see only a single instance of this in our main code base though.

Reply:
I take your point here, I think I'd prefer the rule to be "accessor declarations may not be separated by any expressions that are not declaring accessors". Sadly I don't think that's an option?